src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1625,139 +1526,146 @@ namespace HVACUnitaryBypassVAV {
             CBVAV(CBVAVNum).MaxNoCoolHeatAirMassFlow = RhoAir * CBVAV(CBVAVNum).MaxNoCoolHeatAirVolFlow;
             CBVAV(CBVAVNum).NoCoolHeatOutAirMassFlow = RhoAir * CBVAV(CBVAVNum).NoCoolHeatOutAirVolFlow;
             // set the node max and min mass flow rates
-            Node(MixerOutsideAirNode).MassFlowRateMax = max(CBVAV(CBVAVNum).CoolOutAirMassFlow, CBVAV(CBVAVNum).HeatOutAirMassFlow);
-            Node(MixerOutsideAirNode).MassFlowRateMaxAvail = max(CBVAV(CBVAVNum).CoolOutAirMassFlow, CBVAV(CBVAVNum).HeatOutAirMassFlow);
-            Node(MixerOutsideAirNode).MassFlowRateMin = 0.0;
-            Node(MixerOutsideAirNode).MassFlowRateMinAvail = 0.0;
-            Node(InNode).MassFlowRateMax = max(CBVAV(CBVAVNum).MaxCoolAirMassFlow, CBVAV(CBVAVNum).MaxHeatAirMassFlow);
-            Node(InNode).MassFlowRateMaxAvail = max(CBVAV(CBVAVNum).MaxCoolAirMassFlow, CBVAV(CBVAVNum).MaxHeatAirMassFlow);
-            Node(InNode).MassFlowRateMin = 0.0;
-            Node(InNode).MassFlowRateMinAvail = 0.0;
-            Node(OutNode).Temp = Node(InNode).Temp;
-            Node(OutNode).HumRat = Node(InNode).HumRat;
-            Node(OutNode).Enthalpy = Node(InNode).Enthalpy;
-            Node(CBVAV(CBVAVNum).MixerReliefAirNode) = Node(MixerOutsideAirNode);
+            DataLoopNode::Node(MixerOutsideAirNode).MassFlowRateMax = max(CBVAV(CBVAVNum).CoolOutAirMassFlow, CBVAV(CBVAVNum).HeatOutAirMassFlow);
+            DataLoopNode::Node(MixerOutsideAirNode).MassFlowRateMaxAvail =
+                max(CBVAV(CBVAVNum).CoolOutAirMassFlow, CBVAV(CBVAVNum).HeatOutAirMassFlow);
+            DataLoopNode::Node(MixerOutsideAirNode).MassFlowRateMin = 0.0;
+            DataLoopNode::Node(MixerOutsideAirNode).MassFlowRateMinAvail = 0.0;
+            DataLoopNode::Node(InNode).MassFlowRateMax = max(CBVAV(CBVAVNum).MaxCoolAirMassFlow, CBVAV(CBVAVNum).MaxHeatAirMassFlow);
+            DataLoopNode::Node(InNode).MassFlowRateMaxAvail = max(CBVAV(CBVAVNum).MaxCoolAirMassFlow, CBVAV(CBVAVNum).MaxHeatAirMassFlow);
+            DataLoopNode::Node(InNode).MassFlowRateMin = 0.0;
+            DataLoopNode::Node(InNode).MassFlowRateMinAvail = 0.0;
+            DataLoopNode::Node(OutNode).Temp = DataLoopNode::Node(InNode).Temp;
+            DataLoopNode::Node(OutNode).HumRat = DataLoopNode::Node(InNode).HumRat;
+            DataLoopNode::Node(OutNode).Enthalpy = DataLoopNode::Node(InNode).Enthalpy;
+            DataLoopNode::Node(CBVAV(CBVAVNum).MixerReliefAirNode) = DataLoopNode::Node(MixerOutsideAirNode);
             MyEnvrnFlag(CBVAVNum) = false;
             CBVAV(CBVAVNum).LastMode = HeatingMode;
+            CBVAV(CBVAVNum).changeOverTimer = 0.0;
```

Comment:
initialize changeOverTimer (time when mode changed + new input for min operating time) to 0 on BeginEnvirnFlag